external/partial.patch

```diff
@@ -0,0 +1,114 @@
+diff --git a/tensorflow_serving/util/net_http/server/internal/evhttp_request.cc b/tensorflow_serving/util/net_http/server/internal/evhttp_request.cc
+index c8d0501b..e5bb9700 100644
+--- a/tensorflow_serving/util/net_http/server/internal/evhttp_request.cc
++++ b/tensorflow_serving/util/net_http/server/internal/evhttp_request.cc
+@@ -339,11 +339,19 @@ void EvHTTPRequest::AppendResponseHeader(absl::string_view header,
+ }
+ 
+ void EvHTTPRequest::PartialReplyWithStatus(HTTPStatusCode status) {
+-  NET_LOG(FATAL, "PartialReplyWithStatus not implemented.");
++  bool result =
++      server_->EventLoopSchedule([this, status]() { EvPartialSendReply(status); });
++
++  if (!result) {
++    NET_LOG(ERROR, "Failed to EventLoopSchedule PartialReplyWithStatus()");
++    Abort();
++    // TODO(wenboz): should have a forced abort that doesn't write back anything
```

Comment:
Where does this comes from?

Reply:
This is mimicking existing parts of the code in net_http module. There are non-partial reply methods with similar comments: https://github.com/tensorflow/serving/blob/83c949cd1c997ada425ac2e30510884f16f62b62/tensorflow_serving/util/net_http/server/internal/evhttp_request.cc#L362

I don't think we should care to remove it, let me know if you think otherwise @atobiszei